### ISSUE TITLE
Using raw calls on blocks that are not raw crashes the interpreter

### ISSUE THREAD

**Issue by mbuesch** [mbuesch/awlsim#7]
```
FUNCTION FC 1 : VOID
        VAR_INPUT
                IN0     : INT;
        END_VAR
BEGIN
        L       #IN0
END_FUNCTION


ORGANIZATION_BLOCK OB 1
BEGIN
        UC      FC 1
        CALL SFC 46 // STOP CPU
END_ORGANIZATION_BLOCK
```

```
Traceback (most recent call last):
  File "./awlsim-test", line 578, in <module>
    sys.exit(main())
  File "./awlsim-test", line 575, in main
    return run(inputFile)
  File "./awlsim-test", line 245, in run
    s.runCycle()
  File "/home/mb/develop/git/awlsim/awlsim/core/main.py", line 207, in runCycle
    self.cpu.runCycle()
  File "/home/mb/develop/git/awlsim/awlsim/core/cpu.py", line 882, in runCycle
    self.__runOB(self.obs[1])
  File "/home/mb/develop/git/awlsim/awlsim/core/cpu.py", line 823, in __runOB
    insn.run()
  File "/home/mb/develop/git/awlsim/awlsim/core/instructions/insn_l.py", line 42, in run
    self.cpu.accu1.set(self.cpu.fetch(self.ops[0], {8, 16, 32}))
  File "/home/mb/develop/git/awlsim/awlsim/core/cpu.py", line 1261, in fetch
    return fetchMethod(self, operator, enforceWidth)
  File "/home/mb/develop/git/awlsim/awlsim/core/cpu.py", line 1490, in fetchNAMED_LOCAL
    return self.fetch(self.__translateFCNamedLocalOper(operator, False),
  File "/home/mb/develop/git/awlsim/awlsim/core/cpu.py", line 1201, in __translateFCNamedLocalOper
    interfOp = self.callStackTop.interfRefs[operator.interfaceIndex].resolve(store)
AttributeError: 'CallStackElem' object has no attribute 'interfRefs'
```

It should fail gracefully instead.

**Comment by mbuesch** [comment]
Fixed by 2960e3effc0f3002bf1a633431761303a0ae40ec

**Comment by mbuesch** [comment]
Fix included in awlsim-0.50
